src/clarity-angular/button-group/button-loading.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+
+import {Component} from "@angular/core";
+import {LoadingListener} from "../loading/loading-listener";
+
+@Component({
+    selector: "button[clrLoading]",
```

Comment:
Do this `clrLoading` need to be a bit more specific to buttons, like `clrBtnLoading`?